app/controllers/products_controller.rb

```diff
@@ -40,7 +40,11 @@ def show; end
 
   # GET /products/new
   def new
-    @product = Product.new
+    if CONFIGS[:enable_asset_creation]
+      @product = Product.new
+    else
+      redirect_to '/', notice: { type: 'danger', message: 'Cannot access URL directly.' }
```

Comment:
Can we move the message to the config ? It is something that people might want to override to provide custom re-direct URLs.